pkg/storage/chunk/azure/blob_storage_client.go

```diff
@@ -37,23 +34,18 @@ const (
 
 var (
 	supportedEnvironments = []string{azureGlobal, azureChinaCloud, azureGermanCloud, azureUSGovernment}
-	noClientKey           = azblob.ClientProvidedKeyOptions{}
-	endpoints             = map[string]struct{ blobURLFmt, containerURLFmt string }{
+	endpoints             = map[string]struct{ blobURLFmt string }{
```

Comment:
sdk handles the container url now. We just have to care about which account we're signing into